code/modules/research/destructive_analyzer.dm

```diff
@@ -59,28 +59,25 @@ Note: Must be placed within 3 tiles of the R&D Console
 		return TRUE
 	return FALSE
 
-/obj/machinery/r_n_d/destructive_analyzer/attackby(var/obj/O as obj, var/mob/user as mob)
+/obj/machinery/r_n_d/destructive_analyzer/attackby(var/obj/O, var/mob/user)
 	if(..())
 		return 1
-	if (istype(O, /obj/item) && !loaded_item && !panel_open)
-		if(isrobot(user)) //Don't put your module items in there!
-			if(isMoMMI(user))
-				var/mob/living/silicon/robot/mommi/mommi = user
-				if(mommi.is_in_modules(O,permit_sheets=1))
```

Comment:
This never worked because this machine is using drop_item() and mommis/borgs cannot drop their internal materials.

They can still pick and use stacks, though. Same for mining borgs and their built-in gripper.